package.json

```diff
@@ -51,7 +53,7 @@
     "grunt-contrib-clean": "0.6.0",
     "grunt-contrib-connect": "0.5.0",
     "grunt-contrib-jasmine": "1.0.3",
-    "grunt-contrib-jshint": "0.11.3",
+    "grunt-contrib-jshint": "^0.11.3",
     "grunt-contrib-less": "0.7.0",
     "grunt-contrib-uglify": "0.2.4",
     "grunt-contrib-watch": "0.5.3",
```

Comment:
Could you remove any changes to this file. Discard changes to `package.json` and try installing `grunt-cli` globally with `npm install grunt-cli -g`